backend/dal/Migrations/Initial/Up/PostUp/01-Roles.sql

```diff
@@ -65,6 +65,19 @@ VALUES
     , ''
     , 'migration'
     , ''
+),
+(
+    5
+    , '57726C45-BEE0-4C60-9E93-98E84B4AD136'
+    , 'Undetermined'
```

Comment:
We really want this lol

Reply:
This was the compromise I got out of Doug